src/ResourceManager/Sql/Commands.Sql/Data Sync/Cmdlet/RemoveAzureSqlSyncMember.cs

```diff
@@ -0,0 +1,90 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Sql.DataSync.Model;
+
+namespace Microsoft.Azure.Commands.Sql.DataSync.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to delete a sync member
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlSyncMember", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.High)]
+    public class RemoveAzureSqlSyncMember : AzureSqlSyncMemberCmdletBase
```

Comment:
Do not use ConfirmImact.High. If you want to always prompt the user, use ShouldContinue/Force

Should have an OutputType attribute for the PassThru scenario